app/routes/route_utilities.py

```diff
@@ -0,0 +1,40 @@
+from flask import abort, make_response
+from ..db import db
+
+def validate_model(cls, model_id):
+    model_prefix = 'goal' if cls.__name__.lower() == 'goal' else 'task'
```

Comment:
The idea with functions such as this one is to make it so it can work with any class. This line will only make it work with goal or task and nothing else! Feel free to use `cls.__name__`